docs/topics/gradle/gradle-configure.md

```diff
@@ -0,0 +1,745 @@
+[//]: # (title: Configure your existing project)
+
+To build a Kotlin project with [Gradle](https://docs.gradle.org/current/userguide/getting_started.html), 
+you'll need to add the [Kotlin Gradle plugin](#apply-the-plugin) to your build script file `build.gradle(.kts)` 
+and [configure dependencies](#configure-dependencies) there. As the prerequisite, you need to make sure that you have a Gradle Wrapper in 
+your project or Gradle installed in your system. Consult the [Gradle official documentation](https://docs.gradle.org/current/userguide/getting_started.html#gs:installation) 
```

Comment:
```suggestion
your project or, Gradle installed on your system. Consult the [Gradle official documentation](https://docs.gradle.org/current/userguide/getting_started.html#gs:installation) 
```

Reply:
> As a prerequisite, you need to make sure that you either, have a Gradle Wrapper in your project or, Gradle installed on your system. 

Is it correct to put a comma after "either" and "or"?🤔 Could you please help me with some links to grammar rules on this topic? Would like to learn why to put commas this way and in what cases.

Reply:
I decided to remove this sentence because people, who set up existing projects, already have Gradle. But I would love to understand commas in this sentence anyway:)

Reply:
@sarahhaggarty?:)

Reply:
I don't have a source to hand on this at the moment but I'll keep looking and get back to you on it separately. This is only for instructions, to emphasise the two possible conditions that need to be met.